api/api.go

```diff
@@ -657,13 +661,19 @@ func (api *api) SetNextBackupReminder(backupReminderRequest *BackupReminderReque
 
 var startMutex sync.Mutex
 
-func (api *api) Start(startRequest *StartRequest) error {
+func (api *api) Start(startRequest *StartRequest) (err error) {
+	api.svc.SetStartupError(nil)
```

Comment:
does it need to be on the service? or can it be a variable in the API? it seems it's only used here?

Reply:
No I just thought it belongs there as it is startup information, keeping it in the api is actually better in terms of code as we won't need setters and getters, changing it 👍